### PR TITLE
bitwig-studio3: 3.1.3 -> 3.2.2

### PR DESCRIPTION
**Pull request by tasmo**
###### Motivation for this change

Update Bitwig Studio 3

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 bitwig-studio1.overrideAttrs (oldAttrs: rec {
   name = "bitwig-studio-${version}";
-  version = "3.1.3";
+  version = "3.2.1";
 
   src = fetchurl {
     url = "https://downloads.bitwig.com/stable/${version}/bitwig-studio-${version}.deb";
-    sha256 = "11z5flmp55ywgxyccj3pzhijhaggi42i2pvacg88kcpj0cin57vl";
+    sha256 = "0lcqm4hbkz2d0dmh4ljix0r9cpqpqnjldm3r7f6dg54pcyr9s9c2";
```
Comment (Philipp-M) [inline pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix:9]:
```suggestion
    sha256 = "10zb78n75nbriyjah0m3syv3rv7qwbmj590z24hss7lifa3rs784";
```

File: pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix [hunk 2]
```diff
@@ -2,11 +2,11 @@
 
 bitwig-studio1.overrideAttrs (oldAttrs: rec {
   name = "bitwig-studio-${version}";
-  version = "3.1.3";
+  version = "3.2.1";
```
Comment (Philipp-M) [inline pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix:5]:
```suggestion
  version = "3.2.2";
```

File: pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix (hunk 1 above)
Comment (Philipp-M) [reply pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix:9]: 3.2.2 is out already, so this PR might be updated accordingly, here is the right hash (which might save some time)

**Review by Philipp-M — CHANGES_REQUESTED** [review]

**Comment by wmertens** [comment]
@tasmo can you update with 3.2.2 and I'll merge

**Comment by tasmo** [comment]
@Philipp-M @wmertens Thank you!

**Review by wmertens — APPROVED** [review]

**Pull request merged**